lib/corefoundation.rb

```diff
@@ -2,7 +2,7 @@
 
 module CF
   extend FFI::Library
-  ffi_lib '/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation'
+  ffi_lib [ '/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation', 'libCoreFoundation.so' ]
```

Comment:
@alezummo this shouldn't be necessary, and I'm curious where this `.so` is even coming from. `ffi_lib` should stop at the first loaded library.

Reply:
It’s just for Linux support, in case anyone, like me, wants to use it with https://opensource.apple.com/source/CF/CF-855.14/

Reply:
@alezummo Can you note that in either the code (comment) or commit message?

Reply:
done!